addons/project/static/src/views/project_calendar/calendar_common/project_calendar_common_popover.js

```diff
@@ -0,0 +1,16 @@
+/** @odoo-module **/
+
+import { CalendarCommonPopover } from "@web/views/calendar/calendar_common/calendar_common_popover";
+
+export class ProjectCalendarCommonPopover extends CalendarCommonPopover {
+    get priority() {
+        const record = this.props.record.rawRecord;
+        return (!record.project_id && record.is_favorite) || record.priority;
+    }
+}
```

Comment:
I am sorry, but in agreement with Alexandra, we decided to drop this point (display star before name), because it is requires a lot of customization, as you might have noticed. (Also actually, it would be better to have separate components for project and task, which adds to it)

Reply:
Hello, @auon-odoo removed commit for this point and squashed those 2 commits.